document/FileFormat.mdk

```diff
@@ -2696,31 +2703,318 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|                  | wishes to represent.        |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `assignment`     | Recommended: The            | Recommended: A              | Optional: the names and     |
+|                  | name of the variable        | string representation       | values of the variables     |
+|                  | being assigned to.          | of the value                | participating in the        |
+|                  |                             | assigned to the             | expression on the right-    |
+|                  |                             | target variable.            | hand side of the            |
+|                  |                             |                             | assignment.                 |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `branch`         | Recommended: If the         | Optional: If the branch     | Optional: If the branch is  |
+|                  | target of the branch is a   | is the result of a test,    | the result of a test,       |
+|                  | named label, in which       | in which case, a string     | in which case, the names    |
+|                  | case, the name of           | representation of the       | and values of the variables |
+|                  | the label.                  | Boolean                     | participating in the        |
+|                  | Optional: If the target of  | value of the test           | expression being tested.    |
+|                  | the branch is a known       | condition (`"true"`         | Absent: If the branch is    |
+|                  | address or offset, in       | or `"false"`).              | unconditional.              |
+|                  | which case, a               | Absent: If the branch is    |                             |
+|                  | hexadecimal                 | unconditional.              |                             |
+|                  | string representation of    |                             |                             |
+|                  | that address or offset.     |                             |                             |
+|                  | Absent: If the target of    |                             |                             | 
+|                  | the branch is not           |                             |                             |
+|                  | known.|                     |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `call`           | Recommended: The            | Optional: An array of       | Optional: The names and     |
```

Comment:
>Optional [](start = 51, length = 8)

Ok, just pointing out here that we don't really address the issue of the 'this' pointer, an implied argument. In the instance call below, for example, do we need to worry about capturing the reference to 'example'? Probably not, as for the most part all this data is used to annotate/augment view of source and the use of 'example' is obvious. But what if the checker is trying to tell you that the 'this' pointer is null? Certainly it's no problem to add {"example", "null"} as a variables key/value pair.

example.Call(blah);